src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -312,20 +316,22 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
 
             docker {
                 name '${id}'
-                tags 'latest', 'another'
+                tags 'mytag', 'another'
```

Comment:
This appears to work for me now so I've reverted it back to 'latest'. You will note that the test was always testing for the presence of 'latest', so the intent didn't change :)

Perhaps my machine had a stale state from a previous test run - though having the 'cleanup' block should prevent that.